src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -681,6 +697,15 @@ internal override void SetDefaultSourcePath(string defaultPath)
         }
 
         /// <summary>
+        /// Allow loading assemblies by searching in GAC.
+        /// </summary>
+        /// <returns>Always true, as this is for Full CLR.</returns>
```

Comment:
We don't need to do this for FullCLR. Assembly.Load already takes care of GAC in FullCLR.